clang/include/clang/CConv/ConstraintVariables.h

```diff
@@ -260,6 +263,8 @@ class PointerVariableConstraint : public ConstraintVariable {
   // Get bounds annotation.
   std::string getBoundsStr() { return BoundsAnnotationStr; }
 
+  bool GetIsGeneric(){ return IsGeneric; }
```

Comment:
Why a capital `G` here but just above we use lower-case? Should be consistent, right?